lib/mysql/mysql.go

```diff
@@ -612,31 +614,63 @@ func (c *Connection) decodePacket(body []byte) (PacketInfo, error) {
 	}
 }
 
+// if n, _ := io.Read(body), returns a hex representation of body[:n] that is at most 96 bytes long.
```

Comment:
So is `n` here the maximum length requested?

Reply:
That's poorly worded. I was trying to give a definition of `n` and `body`; `n` being the number of bytes actually read, which can be less than `len(body)` (and `len(body)` is the number of bytes requested)